src/main/java/com/scopely/mapper/JsonDynamoMapper.java

```diff
@@ -61,11 +61,16 @@
  */
 @SuppressWarnings({"WeakerAccess", "unused"})
 public class JsonDynamoMapper {
-    private AmazonDynamoDB amazonDynamoDB;
-    private ObjectMapper objectMapper = new ObjectMapper().registerModule(new Jdk8Module());
+    private final AmazonDynamoDB amazonDynamoDB;
+    private final ObjectMapper objectMapper;
 
     public JsonDynamoMapper(AmazonDynamoDB amazonDynamoDB) {
+        this(amazonDynamoDB, new ObjectMapper().registerModule(new Jdk8Module()));
+    }
+
+    public JsonDynamoMapper(AmazonDynamoDB amazonDynamoDB, ObjectMapper objectMapper) {
```

Comment:
If they don't include the `Jdk8Module`, the `Optional` fields won't work. Maybe that's OK.